eng/testing/tests.mobile.targets

```diff
@@ -133,19 +133,6 @@
       <WasmFilesToIncludeInFileSystem Include="@(ReferenceCopyLocalPaths)" Condition="'%(ReferenceCopyLocalPaths.BuildReference)' == 'true' and !$([System.String]::new('%(ReferenceCopyLocalPaths.Identity)').EndsWith('.resources.dll'))" />
       <WasmFilesToIncludeInFileSystem Include="@(WasmSatelliteAssemblies)" TargetPath="%(WasmSatelliteAssemblies.CultureName)\%(WasmSatelliteAssemblies.Filename)%(WasmSatelliteAssemblies.Extension)" />
       <ExtraAssemblies Include="$(PublishDir)$(AssemblyName).dll" />
-      <!-- we need to preserve these facades for BinaryFormatter tests -->
```

Comment:
Apparently these facades are still needed.

Reply:
I looked at the test failures and it seems only keeping `mscorlib.dll` and `System.Drawing.dll` should be enough, and the latter is even only used by `System.Resources.Extensions.Tests.PreserializedResourceWriterTests.TypeConverterStringResources`.

Reply:
Yeah I'll keep those two. Thanks